packages/mattermost-redux/src/client/client4.ts

```diff
@@ -2651,7 +2651,8 @@ export default class Client4 {
     };
 
     getSystemEmojiImageUrl = (filename: string) => {
-        return `${this.url}/static/emoji/${filename}.png`;
+        const extension = filename.endsWith('.png') ? '' : '.png';
```

Comment:
Is there an inconsistency with some having the extension and others not?

Reply:
no, but the old files used to not have the extension, I can remove this if you prefer

Reply:
0/5, just wanted to understand the context